nautobot/users/tests/test_api.py

```diff
@@ -73,6 +73,123 @@ def setUpTestData(cls):
         Group.objects.create(name="Group 3")
 
 
+class TokenTest(APIViewTestCases.APIViewTestCase):
+    model = Token
+    brief_fields = ["display", "id", "key", "url", "write_enabled"]
+    bulk_update_data = {
+        "description": "New description",
+    }
+
+    def setUp(self):
+        super().setUp()
+
+        tokens = (
+            # We already start with one Token, created by the test class
+            Token(user=self.user),
+            Token(user=self.user),
+        )
+        # Use save() instead of bulk_create() to ensure keys get automatically generated
+        for token in tokens:
+            token.save()
+
+        self.create_data = [
+            {
+                "user": self.user.pk,
+            },
+            {
+                "user": self.user.pk,
+            },
+            {
+                "user": self.user.pk,
+            },
```

Comment:
Since `user` is no longer a serializer field, these inputs will IIRC just be discarded. Perhaps:

```suggestion
            {
                "key": "a" * 40,
            },
            {
                "key": "b" * 40,
            },
            {
                "key": "c" * 40,
            },
```